### PR TITLE
Updated README.md for VM installs

### PR DESCRIPTION
**Pull request by liamfallon**

### CODE REVIEW

File: e2e/provision/README.md [hunk 1]
```diff
@@ -66,6 +73,68 @@ gcloud compute ssh ubuntu@nephio-r1-e2e -- \
...
+
+- Linux Flavour: Ubuntu-22.04-jammy
+- 16 cores
+- 32 GB memory
+- 200 GB disk size
+- default user with sudo passwordless permissions
+
+**Configure a route for Kubernetes**
+
+In some instllations, the IP range used by Kubernetes in the sandbox can clash with the IP address used by your VPN. In such cases, the VM will become unreachable during the sandbox installation. To avoid this situation, add the route below on your VM.
```
Comment (lapentad) [inline e2e/provision/README.md:92]: typo

File: e2e/provision/README.md [hunk 2]
```diff
@@ -66,6 +73,71 @@ gcloud compute ssh ubuntu@nephio-r1-e2e -- \
...
+
+This install has been verified on VMs running on Openstack, AWS, and Azure. 
+
+### VM Prerequisites
+
+Order or create a VM with the following specification:
+
+- Linux Flavour: Ubuntu-22.04-jammy
+- 16 cores
+- 32 GB memory
```
Comment (electrocucaracha) [inline e2e/provision/README.md:86]: Let's standardize the flavor and OS
```suggestion
- Linux Flavour: Ubuntu-20.04-focal
- 8 cores
- 32 GB memory
```

Comment (liamfallon) [reply e2e/provision/README.md]: Done.

File: e2e/provision/README.md [hunk 3]
```diff
@@ -66,6 +73,71 @@ gcloud compute ssh ubuntu@nephio-r1-e2e -- \
...
+
+The following environment variables can be used to configure the installation:
+
+| Variable               | Values           | Default Value | Description                                            |
+| ---------------------- | ---------------- | ------------- | ------------------------------------------------------ |
+| NEPHIO_USER            | userid           | ubuntu        | The user to install the sandbox on (must have sudo passwordless permissions) |
+| NEPHIO_DEBUG           | false or true    | false         | Controls debug output from the install                 |
+| NEPHIO_DEPLOYMENT_TYPE | r1 or one-summit | r1            | Controls the type of installation to be carried out    |
+| RUN_E2E                | false or true    | false         | false: install management cluster only<br>true: install management and  workload clusters |
+| NEPHIO_REPO            | URL              | https://github.com/nephio-project/test-infra.git |URL of repo to use for installation |
```
Comment (electrocucaracha) [inline e2e/provision/README.md:133]:
```suggestion
| RUN_E2E                | false or true    | false         | Determines whether the End-to-End tests are going to be executed |
| NEPHIO_REPO            | URL              | https://github.com/nephio-project/test-infra.git | URL of the repository to be used for installation |
```

**Review by electrocucaracha — COMMENTED** [review]
Last minimal change

File: e2e/provision/README.md (hunk 3 above)
Comment (liamfallon) [reply e2e/provision/README.md]: Done, thanks for correcting the RUN_E2E description, I misunderstood that flag.

**Pull request merged**